source/_reference-a-f.html.md.erb

```diff
@@ -5539,7 +5539,7 @@ Gets the specified dashboard.
 <%= PARAMETERS %> | **include** (optional) -- a comma-separated list of optional elements to include in the response:<ul><li>**source** - the [Source object](#source-object) for any dashboard that was created from another dashboard, if any</li></ul>  
   | **level** (optional): specifies whether new functionality, such as multi-contact data is returned in a backwards-compatible, text format (**level=0**, default), multi-contact data (**level=2**), multi-picklist data (**level=3**), or Metric widget with sheet summary (**level=4**). 
   | **objectValue** (optional) -- when used in combination with a **level** query parameter, includes the email addresses for multi-contact data.
-<%= RETURNS %> | [Dashboard object](#dashboard-object-aka-sight-object)
+<%= RETURNS %> | [Sight object](#sight-object)
```

Comment:
```html
<%= RETURNS %> | [Dashboard object](#dashboard-object)
```
In the referencing too, I think. Presumably this would be even more confusing for viewers of the API if they see objects referred to as "Sight" objects, when they're "Dashboard" objects in the API